### DISCUSSION TITLE
目录如何控制间距

### DISCUSSION THREAD

**Discussion by sfxjh** [stone-zeng/fduthesis#306] [Q&A]
因为目录多出了一两行，自己尝试找issue解决

```tex
\usepackage[titles]{tocloft}
\renewcommand{\cftdot}{$\cdot$}
\renewcommand{\cftdotsep}{1.5}
\setlength{\cftbeforechapskip}{0pt}
\renewcommand{\cftchapleader}{\cftdotfill{\cftchapdotsep}}
\renewcommand{\cftchapdotsep}{\cftdotsep}
```

在 https://github.com/stone-zeng/fduthesis/issues/73 看到了这个解决方式，但有报错：

![image](https://github.com/stone-zeng/fduthesis/assets/22070963/31e1d905-bd72-4896-83ed-8c2f99d366d1)

**Comment by sikouhjw** [comment]
你是不是加了 `subfigure` 宏包？
https://tex.stackexchange.com/questions/336617/command-clofdepth-already-defined-when-using-packages-subfigure-and-tocloft

**Reply by sfxjh** [reply]
是的，目前看起来是单纯报错但没有问题。subfigure宏包有什么替代手段吗？

**Reply by sikouhjw** [reply]
> 是的，目前看起来是单纯报错但没有问题。subfigure宏包有什么替代手段吗？

你但凡看一眼链接都不会问这个问题。
1. `\usepackage[subfigure,titles]{tocloft}`
2. `subfig`, `subcaption`

**Reply by sfxjh** [reply]
嗯好的hhh
1、第一个方式会导致目录标题偏移左侧
2、当时有点急所以没看到subgraph的解决方案
3、可以采用minipage单独排版的方式解决问题
